gtk2_ardour/vca_master_strip.cc

```diff
@@ -394,10 +394,10 @@ VCAMasterStrip::solo_changed ()
 bool
 VCAMasterStrip::vertical_button_press (GdkEventButton* ev)
 {
-	if (ev->button == 1 && ev->type == GDK_2BUTTON_PRESS) {
-		start_name_edit ();
-		return false;
-	}
+	// if (ev->button == 1 && ev->type == GDK_2BUTTON_PRESS) {
+	// 	start_name_edit ();
+	// 	return false;
+	// }
```

Comment:
*If* this old code should remain as comment, it should have a comment to explain why it is removed yet still around.

(But if there is a comment explaining a non-tricky case, people can look in git history and find the code.)